src/impl/tests/test_data.sql

```diff
@@ -0,0 +1,4210 @@
+-- Current Database: `flowkit_ui_backend`
+--
+
+CREATE DATABASE /*!32312 IF NOT EXISTS*/ `flowkit_ui_backend` /*!40100 DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci */ /*!80016 DEFAULT ENCRYPTION='N' */;
+
+USE `flowkit_ui_backend`;
+
+
+--
+-- Dumping data for table `category`
+--
+
+LOCK TABLES `category` WRITE;
+/*!40000 ALTER TABLE `category` DISABLE KEYS */;
+INSERT INTO `category` VALUES (1,'residents','single_location',1,'https://www.flowgeek.org/methods/calculating-mobility-indicators/residents-indicators','Residents','Residents-class indicators describe long-term (monthly) changes in the number of people whose home location is within each area.','{\"fr\": {\"label\": \"R\\u00e9sidents\", \"description\": \"Les indicateurs relatifs aux r\\u00e9sidents d\\u00e9crivent les variations (mensuelles) \\u00e0 long terme du nombre de personnes dont le lieu de r\\u00e9sidence se trouve dans chaque zone.\"}}'),(2,'relocations','flow',2,'https://www.flowgeek.org/methods/calculating-mobility-indicators/relocation-indicators/','Relocation','Relocation-class indicators describe long-term (monthly) changes in the number of people changing their home location between pairs of areas.','{\"fr\": {\"label\": \"Changement de r\\u00e9sidence\", \"description\": \"Les indicateurs relatifs au changement de r\\u00e9sidence d\\u00e9crivent les variations (mensuelles) \\u00e0 long terme du nombre de personnes qui changent de lieu de r\\u00e9sidence entre des paires de zones.\"}}'),(3,'presence','single_location',3,'https://www.flowgeek.org/methods/calculating-mobility-indicators/presence-indicators/','Presence','Presence-class indicators describe short-term (daily) changes in the number of people who are present within each area.','{\"fr\": {\"label\": \"Pr\\u00e9sence journali\\u00e8re\", \"description\": \"Les indicateurs relatifs \\u00e0 la pr\\u00e9sence d\\u00e9crivent les variations \\u00e0 court terme du nombre de personnes pr\\u00e9sentes dans chaque zone.\"}}'),(4,'movements','flow',4,'https://www.flowgeek.org/methods/calculating-mobility-indicators/movement-indicators','Movements','Movements-class indicators describe short-term changes in the number of people who are travelling into, out of and between areas.','{\"fr\": {\"label\": \"Mouvements journaliers\", \"description\": \"Les indicateurs relatifs aux mouvements d\\u00e9crivent les variations \\u00e0 court terme du nombre de personnes qui se d\\u00e9placent vers, depuis et entre les zones.\"}}');
+/*!40000 ALTER TABLE `category` ENABLE KEYS */;
+UNLOCK TABLES;
+
+
+--
+-- Dumping data for table `data_provider`
+--
+
+LOCK TABLES `data_provider` WRITE;
+/*!40000 ALTER TABLE `data_provider` DISABLE KEYS */;
+INSERT INTO `data_provider` VALUES (1,1,'Digicel Haiti','https://www.digicelgroup.com/etc.clientlibs/digicel-redesign/clientlibs/dg_base/resources/digicel-mobile-selected.svg','https://www.digicelgroup.com',NULL);
+/*!40000 ALTER TABLE `data_provider` ENABLE KEYS */;
+UNLOCK TABLES;
```

Comment:
This is generated by mysql's dump machinery rather than handwritten, so I can only speculate really.. My guess would be that there's an explicit lock because you don't want any meddling with the table while the primary keys are switched off?